CentralManagement/ChocoCCM Command Reference/GetCCMOutdatedSoftware.md

```diff
@@ -0,0 +1,51 @@
+# Get-CCMOutdatedSoftware
+
+<!-- This documentation is automatically generated from /Get-CCMOutdatedSoftware.ps1 using GenerateDocs.ps1. Contributions are welcome at the original location(s). -->
+
+Returns all outdated software reported in CCM
+
+## Syntax
+
+~~~powershell
+Get-CCMOutdatedSoftware [<CommonParameters>]
+~~~
```

Comment:
Why is this a separate command from `Get-CcmOutdatedPackage[Member]`?

If anything an `-All` switch on the other command would more than suffice.

Reply:
Explained in above conversation